Boundaries/MKJP/MKJP.json

```diff
@@ -0,0 +1,96 @@
+{
+  "type": "Feature",
+  "properties": {
+    "id": "MKJP",
+    "prefix": [
+      "MKJP"
+    ],
+    "name": "Manley Radar"
+  },
+  "geometry": {
+    "type": "MultiPolygon",
+    "coordinates": [
+      [
+        [
+          [
+            -77.354722,
+            18.972639
+            
+          ],
+          [
+            -77.354722,
+            17.472472
+          ],
+          [
+            -77.087667,
+            17.331194
+          ],
+          [
+            -76.980000,
+            17.292222
+          ],
+          [
+            -76.772750,
+            17.266778
+          ],
+          [
+            -76.599167,
+            17.291111
+          ],
+          [
+            -76.485444,
+            17.332639
+          ],
+          [
+            -76.390556,
+            17.385000
+          ],
+          [
+            -76.306917,
+            17.449361
+          ],
+          [
+            -76.248889,
+            17.509444
+          ],
+          [
+            -76.185583,
+            17.594694
+          ],
+          [
+            -76.148056,
+            17.666111
+          ],
+          [
+            -76.102194,
+            17.804778
+          ],
+          [
+            -76.088972,
+            17.919167
+          ],
+          [
+            -76.095833,
+            18.035000
+          ],
+          [
+            -76.148111,
+            18.207750
+          ],
+          [
+            -76.254472,
+            18.369417
+          ],
+          [
+            -76.376528,
+            18.478472
+          ],
+          [
+            -76.494722,
+            18.542500
+          ]
```

Comment:
As defined in the GeoJSON spec, the first and last coordinate sets must be the same, forming a closed volume. Please amend your file so that this is the case.